taboos.json

```diff
@@ -2050,7 +2050,538 @@
+      {
+        "code": "06022",
+        "text": "This card's constant ability now reads: \"If this card has no charges, remove it from the game.\"",
+        "replacement_text": "Mutated. Bonded (Segment of Onyx). Uses (3 charges). If this card has no charges, remove it from the game.\n[fast] Exhaust Pendant of the Queen and spend 1 charge: Choose a revealed location and select one - move to that location, discover 1 clue at that location, or automatically evade an enemy at that location."
+      },
+      {
+        "code": "06199",
+        "text": "This card's ability gains: \"Remove each committed copy of Three Aces from the game.\"",
+        "replacement_text": "Mutated. Myriad.\nIf you commit 3 copies of Three Aces to a skill test, that test automatically succeeds <i>(do not reveal chaos tokens from the chaos bag)</i>. Then draw 3 cards and gain 3 resources (max once per test). Remove each committed copy of Three Aces from the game."
+      },
+      {
+        "code": "60318",
+        "text": "Instead of \"Shuffle each weakness that was discarded…\", this card's ability now reads: \"After this skill test resolves, draw each weakness discarded by this effect.\"",
+        "replacement_text": "Mutated. After you commit Daredevil to a skill test, discard cards from the top of your deck until you discard a [rogue] skill you can commit to this test. Commit it. After this skill test resolves, draw each weakness discarded by this effect."
+      },
+      {
+        "code": "07003",
+        "text": "This card's [reaction] ability now reads: \"After you discover 1 or more clues at a location with an enemy: Either discover 1 additional clue at that location, or automatically evade that enemy if it is non-[[Elite]]. (Limit once per round.)",
+        "replacement_text": "Mutated.\n[reaction] After you discover 1 or more clues at a location with an enemy: Either discover 1 additional clue at that location, or automatically evade that enemy if it is non-[[Elite]]. (Limit once per round.)\n[elder_sign] effect: +2. If this is an investigation, you may choose any revealed location; you are now investigating as if you were at that location instead."
+      },
+      {
+        "code": "07122",
+        "text": "This card's [reaction] ability now reads: \"When you resolve a [bless] token during a skill test…",
+        "replacement_text": "Mutated. Permanent. Limit 1 [[Covenant]] per deck.\n[reaction] When you resolve a [bless] token during a skill test, exhaust Ancient Covenant: Do not reveal another token as part of this [bless] token's effect."
+      },
+      {
+        "code": "60233",
+        "deck_limit": 0,
+        "text": "Forbidden.",
+        "replacement_text": "Forbidden. Uses (6 secrets). Limit 1 per deck.\n[fast] Spend 1 secret: You get +2 skill value for this test.\n[fast] Spend 2 secrets: Draw 2 cards.\n[fast] Spend 3 secrets: Discover 1 clue at any location.\n][fast] Spend 4 secrets: Deal 3 damage to an enemy engaged with you."
```

Comment:
FYI, prior versions of Double or Nothing said "Mutated. Forbidden...." but the official text is just "Forbidden." I changed Double of Nothing as well.